<?xml version="1.0"?>
<launch>
    <arg name="master_name"     default="$(env ROBOT_MASTER)"/>
    <arg name="surround_robot"  default="robot_2"/>
    <arg name="surround_frame"  default="surround_frame"/>
    <arg name="radius"          default="0.5"/>
    <arg name="speed"           default="30"/><!--15-->

    <!--发布target_frame的实时位置-->
    <node name="tf_publish" pkg="jetauto_multi" type="tf_publish.py" respawn="false" output="screen">
        <param name="radius"        value="$(arg radius)"/>
        <param name="speed"         value="$(arg speed)"/>
        <param name="map_frame"     value="$(arg master_name)/map"/>
        <param name="base_frame"    value="$(arg master_name)/base_footprint"/>
        <param name="target_frame"  value="$(arg surround_frame)"/>
    </node>

    <!--发送控制指令到surround_robot-->
    <node name="tf_listen" pkg="jetauto_multi" type="tf_listen.py" respawn="false" output="screen">
        <param name="base_frame"   value="$(arg surround_robot)/base_footprint"/>
        <param name="target_frame" value="$(arg surround_frame)"/>
        <param name="cmd_vel"      value="/$(arg surround_robot)/jetauto_controller/cmd_vel"/>
    </node>
</launch>
